projects/hslayers/src/components/print/models/text-style.model.ts

```diff
@@ -0,0 +1,12 @@
+import {xPos, yPos} from '../types/XY-positions.type';
+
+export type TextStyle = {
+  fillColor?: string;
+  strokeColor?: string;
```

Comment:
What is this param good for? I can't see it used anywhere in the templates..

Reply:
StrokeColor I use for text stroke style.